documentation/release_notes.markdown

```diff
@@ -8,7 +8,70 @@ canonical: "/puppetserver/latest/release_notes.html"
 [service bootstrapping]: ./configuration.markdown#service-bootstrapping
 [auth.conf]: ./config_file_auth.markdown
 
-For release notes on the previous version of Puppet Server, see [docs.puppet.com](https://docs.puppet.com/puppetserver/2.4/release_notes.html).
+For release notes on versions of Puppet Server prior to Puppet Server 2.5, see [docs.puppet.com](https://docs.puppet.com/puppetserver/2.4/release_notes.html).
+
+## Puppet Server 2.6
+
+Released September 8, 2016.
+
+This is a feature and bug-fix release of Puppet Server. This release also adds an official Puppet Server package for SuSE Enterprise Linux (SLES) 12.
+
+> **Warning:** If you're upgrading from Puppet Server 2.4 or earlier and have modified `bootstrap.cfg`, see the [Puppet Server 2.5 release notes first](#potential-breaking-issues-when-upgrading-with-a-modified-bootstrapcfg) **before upgrading** for instructions on avoiding potential failures.
```

Comment:
Maybe say "modified 'bootstrap.cfg', '/etc/sysconfig/puppetserver', or '/etc/default/puppetserver'" since a change to any of those could cause upgrade problems that the 2.5 notes point out?